content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,99 @@
+# Persisting Experiments
+
+When you run an experiment with `dvc exp run`, it produces temporary objects in
+the Git repository. These are listed with `dvc exp list` and `dvc exp show`.
+
+Although the experiments as the temporary, they can be shared with
+`dvc exp push` and retrieved from other Git repositories with `dvc exp pull`. In
+this section we'll see how to bring them to the standard Git workflow with
+`dvc exp apply` and `dvc exp branch`.
+
+## Bring Experiment results to your workspace
+
+If it's not run with `--temp` or `--queue` flags, `dvc exp run` leaves the final
+artifacts in your workspace. This is for convenience. However if there are more
+than one experiment you have run, like
+
+```dvc
+$ dvc exp run -S my_param=1
+$ dvc exp run -S my_param=2
+$ dvc exp run -S my_param=3
+```
+
+only the final experiment's results are found in the workspace. In this case,
+you may need to bring the previous experiments' results with `dvc exp apply`.
+
+```dvc
+$ dvc exp show --include-params=my_param
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created      ┃     auc ┃ my_param   ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━┩
+│ workspace             │ -            │ 0.61314 │ 3          │
+│ experiments-tag       │ Oct 19, 2020 │ 0.61314 │ 3          │
```

Comment:
What's `experiments-tag`?

Reply:
It's the tag we gave to that commit that contains the experiment. Is it confusing? Let me change it then.

Reply:
Well if it's a real tag in the example repo then OK (unless you want to change it to something more meaningful) but should we at least explain what it is under this image? Maybe as a note (md quote block).

Reply:
> if it's a real tag in the example repo then OK

Actually we don't need to have the real tags here since it's not a tutorial or even get started. You can change the labels and exp names as needed even if they're imaginary.

Reply:
This is probably minor. I just think if someone tries to fully understand the table they may be confused by that revision name. I'd suggest something very obvious like keep `HEAD` or do something like `v1.2.3`. Up to you

Reply:
it's certainly not `HEAD` and `v1.2.3` is not something we'd use in an experimentation setting, there is no linear progress. Changed to `new-experiments` but can replace it if you have a better idea.